components/src/components/relative-growth-advantage.ts

```diff
@@ -45,49 +44,61 @@ export class RelativeGrowthAdvantage extends LitElement {
         args: () => [this.lapis, this.numerator, this.denominator, this.generationTime, this.views] as const,
     });
 
+    heading: string = 'Relative growth advantage';
+
+    getViewTitle(view: View) {
+        switch (view) {
+            case 'line':
+                return 'Line';
+        }
+    }
+
+    getLineChartView(data: RelativeGrowthAdvantageData) {
+        const info = html` <gs-component-info content="Line chart"></gs-component-info>`;
+
+        return html`
+            <gs-component-toolbar .bottomElements=${[info]}>
+                <gs-relative-growth-advantage-chart
+                    .data=${{
+                        ...data.estimatedProportions,
+                        observed: data.observedProportions,
+                    }}
+                ></gs-relative-growth-advantage-chart>
+                <div>
+                    Advantage: ${(data.params.fd.value * 100).toFixed(2)}%
+                    (${(data.params.fd.ciLower * 100).toFixed(2)}% - ${(data.params.fd.ciUpper * 100).toFixed(2)}%)
+                </div>
+            </gs-component-toolbar>
+        `;
+    }
+
+    getViewContent(view: View, data: RelativeGrowthAdvantageData) {
+        switch (view) {
+            case 'line':
+                return this.getLineChartView(data);
+        }
+    }
+
     override render() {
         return this.fetchingTask.render({
-            pending: () => html`
-                <h1>Relative growth advantage</h1>
-                Loading...
-            `,
+            pending: () => html` <gs-component-container heading=${this.heading}> Loading...</gs-component-container> `,
             complete: (data) => {
                 if (data === null) {
-                    return html`
-                        <h1>Relative growth advantage</h1>
-                        <p>No data available.</p>
-                    `;
+                    return html` <gs-component-container heading=${this.heading}>
+                        >No data available.</gs-component-container
+                    >`;
```

Comment:
```suggestion
                    return html` <gs-component-container heading=${this.heading}>
                        No data available.</gs-component-container
                    >`;
```
And the formatting looks quite broken ;)